frontend/apps/os-shell/src/__tests__/legacy/LegacyDeprecationBanner.test.tsx

```
@@ -0,0 +1,181 @@
+ * - Dismiss button hides banner for session
+ * - Banner respects session storage dismissal
+ * - Accessibility: proper ARIA roles and keyboard navigation
+ */
+
+import { fireEvent, render, screen } from '@testing-library/react';
+import { LegacyDeprecationBanner } from '../../components/legacy/LegacyDeprecationBanner';
+
+// Mock sessionStorage
+const mockSessionStorage: Record<string, string> = {};
+beforeEach(() => {
+  Object.keys(mockSessionStorage).forEach((key) => delete mockSessionStorage[key]);
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)